.env.example

```
@@ -11,15 +11,52 @@ APPWRITE_BANK_COLLECTION_ID=
-PLAID_PRODUCTS=
-PLAID_COUNTRY_CODES=
+PLAID_ENV=sandbox
+PLAID_PRODUCTS=auth,transactions,identity
+PLAID_COUNTRY_CODES=US,CA
+
+# PLAID TEST TOKENS (For development/testing only - DO NOT COMMIT!)
+# Generate these from Plaid Dashboard > Sandbox > Generate Token
+PLAID_TEST_ACCESS_TOKEN=
+PLAID_TEST_ACCESS_TOKEN_1=
+PLAID_TEST_ACCESS_TOKEN_2=
+NEXT_PUBLIC_TEST_USER_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_TEST_USER_ID key should go before the PLAID_TEST_ACCESS_TOKEN key

(UnorderedKey)